book/glossary.md

```diff
@@ -83,59 +101,86 @@ __HTML__.
 text and the document object). (Inherits from __Node__.)
 
-__Focus__: the property of an __element__ being the highlight, or "focus",
+__Event__: A way for JavaScript to observe that something has happened on the
+document, and customize its results.
+
+__Focus__: Ahe property of an __element__ being the highlight, or "focus",
 of user interaction, and therefore receiving keyboard events and being
 visually highlighted on the screen.
 
-__IFrame__: a way of embedding a child __document__ within a parent, through
+__Font__: A particular stylistic way of drawing a particular human language to
+computer screens. Arial is one common example for Latin-based langauges.
+
+__IFrame__: A way of embedding a child __document__ within a parent, through
 a rectangular window on the screen reserved for it that participates in the
 layout of the parent.
 
+__Image__: A representation of a picture to draw on a computer screen. An
+HTML element of the same name, for the same purpose.
+
 __Node__: A point in the __DOM__ tree, with parent and child pointers.
 
 __Page__: The conceptual container for a __document__. Unlike its physical
 analogue, a page can have multiple documents (through use of __iframes__).
 
+__Style sheet__: A document resource that contains __CSS__ rules.
 
-Box model
-=========
+__Tag__: The name of a particular type of HTML element, indicating its
+semantic function in the document. Usually comes with special style rules
+and functionality specific to it.
+
+Networking
+==========
+
+__GET__: The mode of HTTP that retrieves a server resource without changing it.
+
+__Cookie__: A piece of persistent, per-site state stored by web browsers
+to enable use cases like user logged-in status for access-controlled content.
 
-![](https://www.w3.org/TR/CSS2/images/boxdim.png)
+__Domain__: The name of a website, used to locate it on the internet.
 
-__Margin rectangle__: The rectangle that encloses the margin (dark dashed line
-in the figure). Margin in the spacing between elements.
+__Path__: The part of a URL after the domain and port.
 
-__Border rectangle__: The rectangle that encloses the border (dark solid line
-in the figure). Borders can have a styled visual representation.
+__Port__: A number after the domain and before the part in a URL, indicating
+a numbered place on that domain with which to communicate.
 
-__Padding rectangle__: The rectangle that encloses the padding (light dashed
-line in the figure). The spacing between the border and the content.
+__POST__: THe mode of HTTP that submits a change to server state and expects
+a newly updated web page in response.
 
-__Content rectangle__: The rectangle that enloses the content (light solid line
-in the figure). Child elements and text are normally enclosed by this
-rectangle.
+__Scheme__: The first part of a URL, indicating which protocol to use for
+communication, such as __HTTP__ or __HTTPS__.
+
+__SSL__: Secure Sockets Layer. An encryption-based protocol that enables
```

Comment:
Should we put the definition term as `TLS/SSL`? Or maybe clarify the distinction in the definition? (SSL is an older protocol, TLS replaces it, all connections now use TLS, though SSL remains common terminology for both.)